test/lib/course_planner/courses/offered_courses_test.exs

```diff
@@ -45,33 +45,39 @@ defmodule CoursePlanner.OfferedCoursesTest do
   describe "find_all_by_user/1" do
     test "coordinators should see all offered courses" do
       user = insert(:coordinator)
-      insert_list(2, :offered_course)
+      term = insert(:term)
+      insert_list(2, :offered_course, term: term)
 
-      courses = OfferedCourses.find_all_by_user(user)
+      terms = OfferedCourses.find_all_by_user(user)
```

Comment:
this became awkward, `OfferedCourses.find_all_by_user` doesn't return offered courses, but terms
I don't think this interface should change

Reply:
shall i change it to something like "find_all_by_user_categorized_by_term" ?
or shall I move it to terms.ex ?

Reply:
I think it would make more sense to move it to terms

Reply:
done